packages/@aws-cdk/aws-ec2/lib/client-vpn.ts

```diff
@@ -0,0 +1,409 @@
+import logs = require('@aws-cdk/aws-logs');
+import cdk = require('@aws-cdk/core');
+import {
+  CfnClientVpnAuthorizationRule,
+  CfnClientVpnEndpoint,
+  CfnClientVpnRoute,
+  CfnClientVpnTargetNetworkAssociation
+} from './ec2.generated';
+import {CIDR_VALIDATION_REGEXES} from "./peer";
+import {ISubnet} from './vpc';
+
+interface IClientAuthenticationRequestOptions {
```

Comment:
Docstring on the type declaration itself as well please. This comment applies to all interfaces and classes.